src/site/resources/css/site.css

```diff
@@ -120,7 +120,7 @@ span.wrapper.inline {
   display: inline;
 }
 
-@media screen and (max-width: 600px) {
+@media screen and (max-width: 823px) {
```

Comment:
@jo2 
https://stackoverflow.com/questions/22672673/apply-css-rule-only-when-device-is-horizontal
> `@media only screen and (orientation: landscape) {`

Reply:
`oriantation` only differs whether the viewport width is bigger than the viewport height or not [see here](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/orientation). That if we'd only use the orientation, it would also trigger for laptops and desktops but not for smartphones in vertical mode.

So if we were to use `orientation` it would have to be in a combination with a `(max)-width`. In this PRs suggestion we handle everything with a width of up to 823px whether it being in horizontal or vertical mode. And in this case we don't want do differ between those modes because we define everything up to 823px as a smartphone or mobile device which should display the mobile version of the page. I don't think we'd gain anything by using an `orientation`